### DISCUSSION TITLE
SealCommit2 Failed: shows ‘error in RPC call to 'Filecoin.SealCommit2': the element is not part of an r-order subgroup’

### DISCUSSION THREAD

**Discussion by verynohair** [filecoin-project/lotus#4641] [Storage Provider]
Seal commit 2 failed when the worker finished sealing a sector. The error shows below:

2020-10-29T05:39:29.265Z        WARN    rpc     go-jsonrpc@v0.1.2-0.20201008195726-68c6a2704e49/handler.go:241error in RPC call to 'Filecoin.SealCommit2': the element is not part of an r-order subgroup
github.com/filecoin-project/filecoin-ffi.SealCommitPhase2
        /opt/upload/all/lotus/extern/filecoin-ffi/proofs.go:403
github.com/filecoin-project/lotus/extern/sector-storage/ffiwrapper.(*Sealer).SealCommit2
        /opt/upload/all/lotus/extern/sector-storage/ffiwrapper/sealer_cgo.go:515
github.com/filecoin-project/lotus/extern/sector-storage.(*LocalWorker).SealCommit2
        /opt/upload/all/lotus/extern/sector-storage/localworker.go:176
reflect.Value.call
        /usr/local/go/src/reflect/value.go:475
reflect.Value.Call
        /usr/local/go/src/reflect/value.go:336
github.com/filecoin-project/lotus/metrics.proxy.func1
        /opt/upload/all/lotus/metrics/proxy.go:61
reflect.callReflect
        /usr/local/go/src/reflect/value.go:564
reflect.makeFuncStub
        /usr/local/go/src/reflect/asm_amd64.s:20
github.com/filecoin-project/lotus/api/apistruct.(*WorkerStruct).SealCommit2
        /opt/upload/all/lotus/api/apistruct/struct.go:1426
reflect.Value.call
        /usr/local/go/src/reflect/value.go:475
reflect.Value.Call
        /usr/local/go/src/reflect/value.go:336
github.com/filecoin-project/go-jsonrpc/auth.PermissionedProxy.func1
        /root/go/pkg/mod/github.com/filecoin-project/go-jsonrpc@v0.1.2-0.20201008195726-68c6a2704e49/auth/auth.go:62
reflect.callReflect
        /usr/local/go/src/reflect/value.go:564
reflect.makeFuncStub
        /usr/local/go/src/reflect/asm_amd64.s:20
github.com/filecoin-project/lotus/api/apistruct.(*WorkerStruct).SealCommit2
        /opt/upload/all/lotus/api/apistruct/struct.go:1426
reflect.Value.call
        /usr/local/go/src/reflect/value.go:475
reflect.Value.Call
        /usr/local/go/src/reflect/value.go:336
github.com/filecoin-project/go-jsonrpc.doCall
        /root/go/pkg/mod/github.com/filecoin-project/go-jsonrpc@v0.1.2-0.20201008195726-68c6a2704e49/handler.go:130
github.co

**Comment by jennijuju** [comment]
This seems like you have a disk corruption, or can be ram too. 

Converting this to a discussion, and please perform some checks on your hardware, and post any further questions!
